app/lib/actions/katello/content/destroy.rb

```diff
@@ -0,0 +1,11 @@
+module Actions
+  module Katello
+    module Content
+      class Destroy < Actions::Base
+        def run
+          ::Katello::Content.delete(input[:katello_content_id])
```

Comment:
.destroy() is more appropriate here.  .delete() bypasses all active record hooks